timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -88,19 +93,20 @@ void BDExchangeMove<RealType>::move(
 
     this->compute_initial_weights(N, d_coords, d_box, stream);
 
+    // All of the noise is generated upfront
+    curandErrchk(templateCurandNormal(cr_rng_, d_quaternions_.data, d_quaternions_.length, 0.0, 1.0));
+    // The d_translation_ buffer is [x,y,z,w] where [x,y,z] are a random translation and w is used for acceptance
```

Comment:
nit: could "w is used for acceptance" be more specific? (e.g. is it a uniform random value between 0 and 1)

Reply:
Expanded the comment in https://github.com/proteneer/timemachine/pull/1234/commits/8981566e878e64e51407adecb7b6fd5320b93c3d